datafusion/optimizer/tests/optimizer_integration.rs

```diff
@@ -198,8 +199,9 @@ fn between_date64_plus_interval() -> Result<()> {
     let plan = test_sql(sql)?;
     let expected =
         "Aggregate: groupBy=[[]], aggr=[[COUNT(Int64(1))]]\
-        \n  Filter: test.col_date64 >= Date64(\"890179200000\") AND test.col_date64 <= Date64(\"897955200000\")\
-        \n    TableScan: test projection=[col_date64]";
+        \n  Projection: \
```

Comment:
It seems there is an additional empty projection here compared to before.

Reply:
Yes, but this (empty) projection is fine, as the aggregate doesn't need any columns.